be/src/storage/column_predicate_rewriter.cpp

```diff
@@ -302,7 +303,23 @@ StatusOr<bool> ColumnPredicateRewriter::_rewrite_expr_predicate(ObjectPool* pool
     size_t value_size = raw_dict_column->size();
     std::vector<uint8_t> selection(value_size);
     const auto* pred = down_cast<const ColumnExprPredicate*>(raw_pred);
-    RETURN_IF_ERROR(pred->evaluate(raw_dict_column.get(), selection.data(), 0, value_size));
+    if (value_size <= pred->runtime_state()->chunk_size()) {
```

Comment:
why not choose to change the parameter type of the evaluate interface. I think chunk_size <= UINT16_MAX may be  not a hard constraint. If we support modifying chunk_size in the future, the current change may leave hidden dangers.